plugins/fake-bot-after.conf

```
@@ -16,13 +16,26 @@
 # Documentation can be found here:
 # https://github.com/coreruleset/fake-bot-plugin
 
-SecRule REQUEST_HEADERS:User-Agent "@pm amazonbot applebot bingbot linkedinbot facebookbot facebookcatalog facebookexternalhit googlebot twitterbot" \
+SecRule TX:FAKE-BOT-PLUGIN_WHITELIST_BROKEN_APPLE_DEVICES "@streq 1" \
```

I wouldn't call the variable "...broken".  From what I've read, this appears to be mostly Apple's iMessage when it tries to retrieve a preview, but Viber apparently does the same (https://webmasters.stackexchange.com/questions/137914/spike-in-traffic-from-facebot-twitterbot-user-agent). So this is actually more a symptom of messengers, trying to please users.
```suggestion
SecRule TX:FAKE-BOT-PLUGIN_WHITELIST_MESSENGER_USER_AGENTS "@streq 1" \
```

---

I would call it broken as such behavior is... broken.

---

As you wish :D

---

@theseion Just tested Viber and it is not doing it:
`User-Agent: Mozilla/5.0 (Linux; Android 14; 2306EPN60G Build/UP1A.230905.011; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/131.0.6778.135 Mobile Safari/537.36 Viber/24.3.1.0`

So the problem is realted only with Apple-shit.